js/services/vCardProperties.js

```diff
@@ -91,15 +91,16 @@ angular.module('contactsApp')
 		impp: {
 			multiple: true,
 			readableName: t('contacts', 'Instant messaging'),
-			template: 'text',
+			template: 'username',
 			defaultValue: {
 				value:[''],
-				meta:{type:['HOME']}
+				meta:{type:['SKYPE']}
 			},
 			options: [
-				{id: 'HOME', name: t('contacts', 'Home')},
-				{id: 'WORK', name: t('contacts', 'Work')},
-				{id: 'OTHER', name: t('contacts', 'Other')}
+				{id: 'IRC', name: t('contacts', 'Irc')},
```

Comment:
I would say, that those don't need to be translated. Also I would write "IRC", because it is an abbreviation.